spec.md

```diff
@@ -77,55 +82,59 @@ progressively greater amount of knowledge."
 **Suggestions:**
 - May capture third and fourth depth headings. If it is a long ToC, these are optional.
 
-#### Security
-**Status**: Optional.  
+### Security
+**Status**: Optional.
 **Requirements:**
-- May go here if visibility of security section is important. Otherwise, should be in [Extra Sections](#extra-sections).
+- May go here if it is important to highlight security concerns. Otherwise, it should be in [Extra Sections](#extra-sections).
 
-#### Background
-**Status:** Optional.  
+### Background
+**Status:** Optional.
 **Requirements:**
 - Cover motivation.
 - Cover abstract dependencies.
 - Cover intellecutal provenance: A `See Also` section is also fitting.
 
-#### Install
+### Install
 **Status:** Required by default, optional for doc modules.
 **Requirements:**
 - Code block illustrating how to install.
 
-Subsections:
-- `Dependencies`. Required if necessary.
+**Subsections:**
+- `Dependencies`. Required if there are dependencies.
 - `Updating`. Optional.
 
 **Suggestions:**
-- Link to prerequisite sites for language: [npmjs](https://npmjs.com), [godocs](https://godoc.org), etc.
-- Include any system-specific information needed for Installation.
-- Subsection for dependencies needed for install to work.
+- Link to prerequisite sites for programming language: [npmjs](https://npmjs.com), [godocs](https://godoc.org), etc.
+- Include any system-specific information needed for installation.
 - If there is no code in the module - for instance, a document-based module - this section is not required.
 
-#### Usage
+### Usage
 **Status:** Required by default, optional for doc modules.
 **Requirements:**
 - Code block illustrating common usage.
 - If CLI compatible, code block indicating common usage.
-- If importable, code block indicating both import functionality and usage. 
+- If importable, code block indicating both import functionality and usage.
 
-Subsections:
+**Subsections:**
 - `CLI`. Required if CLI functionality exists.
-- If relevant, point to a runnable file for the usage code.
 
 **Suggestions:**
 - Cover basic choices that may affect usage: for instance, if JavaScript, cover promises/callbacks, ES6 here.
+- If relevant, point to a runnable file for the usage code.
 - If there is no code in the module - for instance, a document-based module - this section is not required.
 
-#### Extra Sections
-**Status**: Optional.  
+### Extra Sections
+**Status**: Optional.
 **Requirements:**
 - None.
 
-#### API
-**Status:** Optional.  
+**Suggestions:**
+- Does not require its own title.
```

Comment:
Should specify that this encapsulates other possible sections, which must have a title.